storage/registers.go

```diff
@@ -0,0 +1,33 @@
+package storage
+
+import (
+	"io"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// Registers defines methods for the register index.
+type Registers interface {
+	RegisterReader
+	RegisterWriter
+	io.Closer
```

Comment:
```suggestion
```

Reply:
wait what 😮 I guess I was dreaming it then... sorry, will remove.